src/pages/place.tsx

```diff
@@ -30,14 +31,34 @@ export default function PlacePage() {
   const l = useFormatMessage()
   const track = useTrackContext()
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const [account, accountState] = useAuthContext()
   const [share] = useShareContext()
   const location = useLocation()
-  const params = new URLSearchParams(location.search)
 
-  const [placeRetrived] = usePlaceId(params.get("id"))
+  const params = useMemo(
+    () => new URLSearchParams(location.search),
+    [location.search]
+  )
+
+  const [placeRetrived, placeRetrivedState] = usePlaceFromParams(params)
+
+  const placeMemo = useMemo(
+    () => (!placeRetrived ? [[]] : [[placeRetrived]]),
+    [placeRetrived]
+  )
+
+  useEffect(() => {
+    if (
+      placeRetrived &&
+      placeRetrived.base_position &&
```

Comment:
Is it possible for `base_position` to be empty if the places exist? If it is, we should look for a more robust alternative to represent the place URL